WORKSPACE.bazel

```diff
@@ -84,22 +87,12 @@ llvm_toolchain(
     link_libs = {
         "": ["--driver-mode=g++"],
         "linux-x86_64": [
-            "-stdlib=libc++",
+            #"-stdlib=libc++",
```

Comment:
This works for me locally if we link against `libstdc++` which isn't great since we built with headers from `libc++`.

Otherwise, I'm seeing issues like:
```
==================== Test output for //huffman/test:table_from_frequencies_test:
/home/oliver/.cache/bazel/_bazel_oliver/adb62031b64cc76568fe096c0cc5a5f9/sandbox/linux-sandbox/221/execroot/__main__/bazel-out/k8-fastbuild/bin/huffman/test/table_from_frequencies_test.runfiles/__main__/huffman/test/table_from_frequencies_test: symbol lookup error: /lib/x86_64-linux-gnu/libpthread.so.0: undefined symbol: __libc_pthread_init, version GLIBC_PRIVATE
```
likely due to how `libc++` was built.